cmd/exporter.go

```diff
@@ -79,7 +79,7 @@ func main() {
 	}
 	err = newCollector.Attach()
 	if err != nil {
-		klog.Fatalf("%s", fmt.Sprintf("failed to attach : %v", err))
+		klog.Infof("%s", fmt.Sprintf("failed to attach : %v", err))
```

Comment:
@rootfs , but we may need to think twice before merge. 
for this pr, we can accept with info level for now, if bcc libs not found on host instance. for long term considering, 
Q: if host instance/vm without bcc then what should we do?
Q: should we provide any by default approach to ensure restful api works?

Reply:
The api will work if bpf program is not running. Which means that the code will execute without performance counters.

Reply:
the collector loop will not be able to read anything if eBPF is not loaded. That'll bring more to debug. I suggest:
- add build flags for panic()/fatalf and only build with them for CI test
- investigate non-bcc alternatives in a separate stream if necessary

Reply:
> the collector loop will not be able to read anything if eBPF is not loaded. That'll bring more to debug. I suggest:
> 
> * add build flags for panic()/fatalf and only build with `Infof` for CI test
> * investigate non-bcc alternatives in a separate stream if necessary

In my point of view, I suppose this PR is just a bug fix, and we can do further thing through #282 or a new ticket.
I am not sure if #282 , related with #192  and blocks #162 as the pod in CI will not in running status before this fix.

And once again, I hope we can make kepler running in MVC pattern for restapi.
which 
- return number of 0/-1/-NaN instead of a panic as restful response.
- use adopter/strategy pattern adopt with different eBPF libraries and CPU architectures.

or we have an agreement that if kepler running on vm/instance which fails with eBPF loading there should be a panic?
if so, which means for CI we may have to install bcc at ubuntu based github action agent.